### ISSUE TITLE
TONE_CHIBIOS

### ISSUE THREAD

**Issue by zenuavsolutions** [ArduPilot/ardupilot#9990]
### Feature request

**Is your feature request related to a problem? Please describe.**
professional users want to be able to adapt or brand the sounds of Ardupilot without having to compile code

**Describe the solution you'd like**
why not put the sound strings into a text file on SD card?

**Describe alternatives you've considered**
a default set could be in compiled in fw and as string on SD Card, if there are differences present, the file on SD Card could be loaded.

**Platform**
[X] All
[  ] AntennaTracker
[  ] Copter
[  ] Plane
[  ] Rover
[  ] Submarine

**Comment by IamPete1** [comment]
You can do custom tones from scripting, of course that will not disable the existing tones. SD card is not a great solution as they would be lost if the SD card is changed. Of course you could load the file into ROMFS, but at that point you might as well just run a custom build. If we were to implement this I'm not sure it is worth the significant extra flash cost to enable this for all users, so this would end up as a compile time option and at that point you have to do a custom build anyway. 

We could make it easier to change the tones at compile time with defines. But it is not complicated at the moment, just change  them here:

https://github.com/ArduPilot/ardupilot/blob/a39a948c57b53fb4c36ace731c1a40fea9233638/libraries/AP_Notify/ToneAlarm.cpp#L36

You can use https://firmware.ardupilot.org/Tools/ToneTester/ to test tones.

In conclusion I would quite like to close this.....
